summernote-cleaner.js

```diff
@@ -142,7 +143,7 @@
             }
             if (text) {
               /*clean the text first to prevent issues where code view wasn't updating correctly*/
-              var cleanedContent = cleanPaste(text, options.cleaner.badTags, options.cleaner.keepTagContents, options.cleaner.badAttributes, options.cleaner.imagePlaceholder, isHtmlData);
+              var cleanedContent = cleanPaste(text, options.cleaner.badTags, options.cleaner.keepTagContents, options.cleaner.badAttributes, option.cleaner.keepImages, options.cleaner.imagePlaceholder, isHtmlData);
```

Comment:
Please change "option.cleaner.keepImages" and add the s to "options", thank you.

Reply:
Ah thanks well spotted. Updated